spec/biker_spec.rb

```diff
@@ -0,0 +1,70 @@
+require './lib/ride'
+require './lib/biker'
+
+include Ride
+
+class Biker
+  attr_reader :name,
+              :acceptable_terrain,
+              :max_distance
+
+  def initialize(name, acceptable_terrain)
+    @name = name
+    @acceptable_terrain = acceptable_terrain
+    @max_distance = max_distance 
+  end
+
+  it 'has a name' do
+    biker = Biker1.new("Kenny")
```

Comment:
`Biker.new` --> be careful to run these tests to find errors before moving on to the next test! Biker1 is not the name of the class, but rather Biker